src/components/dialogs/load-creation-dialog.js

```diff
@@ -176,16 +180,18 @@ const LoadCreationDialog = ({
                 connectivity.busOrBusbarSection.id,
                 editData ? true : false,
                 editData ? editData.uuid : undefined
-            ).catch((errorMessage) => {
-                displayErrorMessageWithSnackbar({
-                    errorMessage: errorMessage,
-                    enqueueSnackbar: enqueueSnackbar,
-                    headerMessage: {
-                        headerMessageId: 'LoadCreationError',
-                        intlRef: intlRef,
-                    },
+            )
+                .then(() => dispatch(modificationCreationInProgress()))
```

Comment:
Need to be added on every equipment creation ?

Reply:
good catch. Ofc it should, went a little fast on that